src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    public void doWork() {
+        System.out.println("Bulldozer has started working.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer has stopped working.");
```

Comment:
Similarly, the stop message should also be more informative. It's good practice to include the class name in the message to clearly indicate which `Machine` has stopped working.